src/pages/components/index.tsx

```diff
@@ -0,0 +1,7 @@
+import type { NextPage } from "next";
+
+const Components: NextPage = () => {
+  return <></>;
+};
+
+export default Components;
```

Comment:
命名の方針次第ではあるんですが、
ページ用のコンポーネントの suffix に `Page` をいれてみるのはどうでしょう？

cf.
https://github.com/vercel/next.js/blob/canary/examples/with-typescript/pages/about.tsx

```suggestion
const ComponentsPage: NextPage = () => {
  return <></>;
};

export default ComponentsPage;
```